decentraland/unity-renderer
# Feat/avatar particle effects

## What does this PR change?

This branch adds visual particle effects to avatar expressions, and a subtle smoke effect when running/jumping/landing.
Currently, these expressions generate particles:
- Clap
- Throw money
- Kiss

## How to test the changes?

1. Go to *playtest link here*.
2. Press V to exit first-person mode.
3. Press B to bring up the expression menu.
4. Try clapping, throwing money or the kiss.
5. Notice the particles that appear. They should be spawned in a sensible location on the avatar.

Remote avatars will also generate particles, so follow some people around and wait for them to do any expressions to test this.
To test the new smoke particles, just run around and jump. They will spawn at the avatars feet.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
